lib/webview.dart

```diff
@@ -16,7 +16,12 @@ class _ScreenerAppState extends State<ScreenerApp> {
   late WebViewController controller;
   final _razorpay = Razorpay();
   var options = {};
-  late bool isLoading = false;
+  late final String _screenerHomeUrl =
+      widget.debug ? "http://10.0.2.2:8000" : 'https://www.screener.in/';
+  late String paymentUrl = "'$_screenerHomeUrl/payment/capture/'";
```

Comment:
this will add / twice before payment. Remove / at the end of screener.in in the above line